packages/taquito-core/src/taquito-core.ts

```diff
@@ -5,3 +5,4 @@
 
 export * from './error/errors';
 export * from './error/common';
+export * from './interface/http_status_code';
```

Comment:
curious why its called interface?
I was thinking maybe `constants` since its part of something that should never change

Reply:
it's updated as constants